conferences/2020/general.json

```diff
@@ -292,16 +283,30 @@
     "country": "France",
     "twitter": "@DevoxxFR"
   },
+  {
+    "name": "Emerging Technologies for the Enterprise",
+    "url": "https://2020.phillyemergingtech.com/",
+    "startDate": "2020-04-14",
+    "endDate": "2020-04-15",
+    "city": "Online",
+    "country": "Online"
+  },
   {
     "name": "AllTheTalks",
     "url": "http://AllTheTalks.online",
     "startDate": "2020-04-15",
     "endDate": "2020-04-15",
     "city": "Online",
     "country": "Online",
-    "twitter": "@allthetalksconf",
-    "cfpUrl": "https://www.allthetalks.online/speak",
-    "cfpEndDate": "2020-04-01"
+    "twitter": "@allthetalksconf"
+  },
+  {
+    "name": "MagnoliaJS",
```

Comment:
Seems that the event is fully focused on JavaScript. Let's move it to JavaScript?